binders/kafka-binder/pom.xml

```diff
@@ -53,33 +53,6 @@
 					</exclusion>
 				</exclusions>
 			</dependency>
-			<dependency>
-				<groupId>org.apache.kafka</groupId>
-				<artifactId>kafka_2.13</artifactId>
-				<version>${kafka.version}</version>
```

Comment:
We should remove this property altogether and let Boot manage the versions of all dependencies.

Reply:
Sure enough, the SB BOM covers all of them in [2.6.x](https://github.com/spring-projects/spring-boot/blob/6d80b601b69f02059bcda618049039a0cbea550c/spring-boot-project/spring-boot-dependencies/build.gradle#L1057) as well as [2.7.x](https://github.com/spring-projects/spring-boot/blob/392aec4a789ece4341cb3c8e7fac13b474874d65/spring-boot-project/spring-boot-dependencies/build.gradle#L1088). 

I have removed the `kafka.version` and the `dependencyManagement` altogether and tested against `2.7x` and `2.6x` and all is well. 

This eases the concerns of whether or not the `kafka_213` is required or not. However, for future reference, @garyrussell do you know if that jar is required any longer? (from my concern in original description).

Reply:
Yes, it is needed by the embedded broker code (`TestUtils.createServer()`).

Reply:
Right, but other than for testing, it is not needed by consumers of SCSt-kafka. Correct? 

> My understanding is that the only reason for having this lib around besides for @EmbeddedKafka is for the older Kafka clients. I do not think this is valid any longer w/ the versions of Kafka that spring-kafka supports

Reply:
I am not sure what you mean; it has nothing to do with the clients; it is required by the embedded broker.

https://github.com/spring-projects/spring-kafka/blob/51db0b6ecca9514686b4c138de69a1d9efe80664/spring-kafka-test/src/main/java/org/springframework/kafka/test/EmbeddedKafkaBroker.java#L350

The broker uses some common code from the clients, so everything has to be the same version.

Reply:
To close this thread up, we chatted offline and the summary was this for the `kafka_213` scala jar:

> The one classified as `test` has never been used by anything other than the embedded broker; 
The one not classified as `test` might have been used by very early versions of the clients but now all common code (used by both the broker and clients) has been moved to Java and is located in kafka-clients jar

IOW - it would be safe to remove the one not classified as `test`.